<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimitSettingWnd</class>
 <widget class="QDialog" name="LimitSettingWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Max/Min setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>13</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="m_minSpeedEdit">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QDoubleSpinBox" name="m_maxSpeedEdit">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Frequency:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="m_minFreqEdit">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDoubleSpinBox" name="m_maxFreqEdit">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CPitchSpinBox" name="m_minPitchEdit">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="CPitchSpinBox" name="m_maxPitchEdit">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPitchSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>PitchSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_minSpeedEdit</tabstop>
  <tabstop>m_maxSpeedEdit</tabstop>
  <tabstop>m_minFreqEdit</tabstop>
  <tabstop>m_maxFreqEdit</tabstop>
  <tabstop>m_minPitchEdit</tabstop>
  <tabstop>m_maxPitchEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LimitSettingWnd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LimitSettingWnd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
